components/Markdown.css

```diff
@@ -39,14 +39,46 @@
     margin-left: 2rem;
   }
 
-  p > code {
-    background-color: var(--neutral-100);
-    outline: 0.15em solid var(--neutral-100);
+  p>code {
+    background-color: light-dark(hsl(230, 1%, 98%), hsl(220, 13%, 18%));
+    outline: 0.15em solid light-dark(hsl(230, 1%, 98%), hsl(220, 13%, 18%));
   }
 
   pre:has(code) {
-    outline: 1rem solid var(--neutral-100);
-    background-color: var(--neutral-100);
+    outline: 1rem solid light-dark(hsl(230, 1%, 98%), hsl(220, 13%, 18%));
+    background-color: light-dark(hsl(230, 1%, 98%), hsl(220, 13%, 18%));
     overflow: auto;
   }
```

Comment:
**suggestion:** What are these colors? Let's try to use the same bg colors as before if it still looks good. Otherwise at least make them variables.